src/WebJobs.Script.Grpc/Channel/GrpcWorkerChannel.cs

```diff
@@ -247,7 +247,7 @@ public void SetupFunctionInvocationBuffers(IEnumerable<FunctionMetadata> functio
             _functions = functions;
             foreach (FunctionMetadata metadata in functions)
             {
-                _workerChannelLogger.LogDebug("Setting up FunctionInvocationBuffer for function:{functionName} with functionId:{id}", metadata.Name, metadata.GetFunctionId());
+                _workerChannelLogger.LogDebug("Setting up FunctionInvocationBuffer for function: '{functionName}' with functionId: '{functionId}'", metadata.Name, metadata.GetFunctionId());
```

Comment:
Was chatting with @pragnagopa about the changes in this PR and my concern is that changing these placeholder names will result in potential breaking changes. If these are logs going into AI, any queries using the previous names will be broken, which could also impact alerts. Probably not something likely with this particular change, but we need to be careful with these kinds of changes in the future. Thoughts? 

@brettsam

Reply:
You're correct. While I'm not sure this change will have a huge impact anywhere... but it would be safer to leave it as-is.

Reply:
Logging changes in this PR are limited to RPC flow. Impact is minimal. Unless either of you have strong objections, I would like to move ahead with the change. Let me know.